requirements.txt

```diff
@@ -1,15 +1,16 @@
 click >= 7.0
 cloudpickle >= 1.5.0
 dask == 2022.10.2
-jinja2
+cytoolz >= 0.10.1
```

Comment:
I don't know that we want to add `cytoolz` as a hard dependency here. I'm curious what the motivation behind this change is

Reply:
Added it because it was listed as a runtime dependency in the recipe:

https://github.com/dask/distributed/blob/5b6351a6800b0cad76d599962d3394677c791d1a/continuous_integration/recipes/distributed/meta.yaml#L34

Though fi there's a reason that this should be included in the conda recipe but not here I can remove

Reply:
Yeah, we now build _97_ wheels for `cytoolz`, so it should be reliable to install via pip.

Reply:
> I think it was historically difficult to install cytoolz as it required a compiler to be installed

Ah okay, I had just checked to see if it was available on PyPI but didn't realized that older versions only publish the source distribution.

> we now build 97 wheels for cytoolz

It looks like wheels for cpython 3.8-3.10 are only available for 0.12.0 and up, so maybe we should hold off on adding here until distributed requires `cytoolz >=0.12.0`?

Reply:
> we now build 97 wheels

!

> It looks like wheels for cpython 3.8-3.10 are only available for 0.12.0 and up, so maybe we should hold off on adding here until distributed requires cytoolz >=0.12.0?

+1 for holding off for now. Let's keep as an optional dependency, but not a hard requirement